app/controllers/interviews_controller.rb

```diff
@@ -43,8 +53,19 @@ def destroy
 
   private
 
+  def no_conflicts(members, new_interview)
+    members.each do |member|
+      @conflicting_interviews = Interview.select("interviews.id").joins(:user_interviews).where(scheduled_time: new_interview.scheduled_time .. new_interview.end_time)
```

Comment:
suppose there is an inteview with start and end as 5pm and 7pm 
what if i schedule an interview with start and end as 5:30pm and 6:30 pm

Reply:
That's a case I missed, will rectify soon.

Reply:
Resolved in https://github.com/viditjain5598/BetterInterviews/pull/4